about.html

```diff
@@ -0,0 +1,81 @@
+<html>
+<meta charset="utf-8">
+<meta name="viewport" content="width=device-width, initial-scale=1">
+
+<head>
+    <title>About - CEC </title>
+    <link rel="stylesheet" href="css/bootstrap.min.css">
+</head>
+
+<body>
+    <header>
+        <nav class="navbar navbar-expand-lg navbar-dark bg-dark">
+            <div class="container-fluid">
+                <a class="navbar-brand" href="#">Navbar</a>
```

Comment:
Two things here:

- Why keep "navbar" and not put your name or page name?
- This sould redirect to index.html. This is what the user is expecting.